chromadb/config.py

```diff
@@ -225,7 +225,7 @@ def stop(self) -> None:
     def reset_state(self) -> None:
         """Reset the state of this system and all constituents in reverse dependency order"""
         if not self.settings.allow_reset:
-            raise ValueError("Resetting is not allowed by this configuration")
+            raise ValueError("Resetting is not allowed by this configuration (to enable it, your environment variables must include `ALLOW_RESET=TRUE`)")
```

Comment:
I think we'd like something that is not only offering environment variables as a solution. As per #917 - you can also set allow_rest to True.

Reply:
I agree with this.

Reply:
ALLOW_RESET=TRUE doesn't seem to work for me, is it working?